### PR TITLE
Added support for importing from DNS TXT records

### PR DESCRIPTION
**Pull request by awnumar**
This PR adds support for importing from DNS TXT records.

The test records can be found here:

```
importfrom-hello.libeclipse.me
importfrom-bye.libeclipse.me
```

### CODE REVIEW

File: importfrom/__init__.py [hunk 1]
```diff
@@ -27,10 +31,7 @@ def pastebin(url):
 
 def twitter(url):
     page = BeautifulSoup(request(url), "html.parser")
-    try:
-        text = page.body.find('div', attrs={'class':'js-tweet-text-container'}).text.strip()
-    except AttributeError:
-        raise Exception('No tweet found at that URL.')
+    text = page.body.find('div', attrs={'class':'js-tweet-text-container'}).text.strip()
```
Comment (tobbbles) [inline importfrom/__init__.py:34]: What if the body has no div?

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='importfrom',
-      version='1.2.3',
+      version='1.3.0',
```
Comment (mnzt) [inline setup.py]: This seems pretty backward compatible to me

File: importfrom/__init__.py [hunk 3]
```diff
@@ -40,6 +41,11 @@ def gist(url):
     return magic(request(url))
 
 
+def dns(domain):
+    response = json.loads(request('https://dns.google.com/resolve?name=%s&type=TXT' % domain))
+    return magic(response['Answer'][0]['data'][1:-1])
```
Comment (tobbbles) [inline importfrom/__init__.py:46]: What if the domain has multiple dns records?

**Review by tobbbles — COMMENTED** [review]
You should update the README to include instructions for setting txt records

File: setup.py (hunk 2 above)
Comment (awnumar) [inline setup.py:4]: It is. Lrn2semver.

File: importfrom/__init__.py (hunk 3 above)
Comment (awnumar) [reply importfrom/__init__.py:46]: Yeah that's in the readme

File: importfrom/__init__.py (hunk 1 above)
Comment (awnumar) [reply importfrom/__init__.py:34]: 404 is caught earlier. If it's not a tweet then it's the devs fault

**Pull request merged**